tests/custom-resource-edit-selection-test.js

```diff
@@ -0,0 +1,172 @@
+import { beforeEach, afterEach, describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import { describeApplication } from './helpers';
+import ResourceEditPage from './pages/resource-edit';
+import PackageSearchPage from './pages/package-search';
+
+describeApplication('CustomResourceHoldingSelection', () => {
+  let provider,
+    providerPackage,
+    resource;
+
+  beforeEach(function () {
+    provider = this.server.create('provider', {
+      name: 'Cool Provider'
+    });
+
+    providerPackage = this.server.create('package', 'withTitles', {
+      provider,
+      name: 'Star Wars Custom Package',
+      contentType: 'Online',
+      isCustom: true
+    });
+
+    let title = this.server.create('title', {
+      name: 'Hans Solo Director Cut',
+      publicationType: 'Streaming Video',
+      publisherName: 'Amazing Publisher',
+      isTitleCustom: true
+    });
+
+    title.save();
+
+    resource = this.server.create('resource', {
+      package: providerPackage,
+      title,
+      url: 'https://www.frontside.io',
+      isTitleCustom: true,
```

Comment:
No need to set `isTitleCustom` here, since the resource will inherit that from the title.

Reply:
Yep. That is true